modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -0,0 +1,116 @@
+//Tomas how will they best implement this?
+
+== Guidelines for Implementing Multiple Client Providers
+
+Before you implement multiple client providers in Anypoint Platform, ensure that you carefully review the following guidelines and best practices:
+
+* When changing the client provider for an existing API instance, ensure that the API:
+** Has no contracts
+** Has no client provider-related policies applied, such as PingFederate and OpenID policies.
+* When promoting and importing an API:
+
+* If you assign an external client provider to an environment, all existing APIs in that environment display Anypoint Platform in the UI by default. 
++
+You must then reassign the appropriate external client provider for that API.
+* After you assign an external client provider to an API, you cannot reassign the client provider back to the default native Anypoint Platform credential provider.
+* If you remove a client provider from an environment, all existing APIs and client applications using that client provider continue to work.
+* If you delete an client provider from a master organization, all existing APIs and client applications using that client provider default to the native Anypoint Platform client provider. 
++
+All contracts remain intact.
+
+
+== Configuring Multiple Client Providers User Flow
+
+Based on the use case explained in the previous section, Good Weather uses the following flow to implement multiple external client providers:
+
+image::multiple-IdP-user-flow.png[User Flow for Configuring Multiple External Client Providers]
+
+=== Tasks for Implementing Multiple Client Providers Based on Roles
+
+Based on your role in your organization and your requirements, you might need to perform the following set of tasks to implement multiple client providers:
+
+[%header,cols="20a,40a,20a,50a"]
+|===
+| Role | Task | Where to Perform | Notes
+| Business Group Administrator | Assign multiple IdPs to an environment and also assign a default IdP. | Access Management | Using multiple IdPs provides flexibility to choose an IdP in API configuration inside API Manager, which can be useful when multiple teams are using the same environment for building APIs, but still want to use separate IdPs. This also helps in use cases when separate API instances are created for internal and external consumers, and separate IdPs based on a consumer is required. 
+| Business Group Administrator | Be notified about any possible interruption to API consumers when I am about to change any IdP configuration.  | Access Management | None
+| Master organization Administrator | Change the default IdP across all environments and across all business groups. | Access Management | Resetting the default IDP updates all environments across all business groups to a selected IDP, without affecting existing client applications. This way, when a new IdP is created for the organization, all environments are updated with just one change.
```

Comment:
"Change the default IdP across all environments and across all business groups." This is not something that can be done AFAIK.

Reply:
commented out